pyke/lightcurve.py

```diff
@@ -140,3 +140,28 @@ def detrend(self):
 class ArcLengthDetrender(Detrender):
     def detrend(time, flux):
         pass
+
+
+class EverestDetrender(Detrender):
```

Comment:
We're not actually implementing Everest, so I would call this e.g. `SimplePixelLevelDetrender`.  The docstring should of course give credit to the Wang and Luger papers that explain the idea, and mention everest!